src/openvic-simulation/research/Invention.cpp

```diff
@@ -2,6 +2,7 @@
 
 #include "openvic-simulation/economy/BuildingType.hpp"
 #include "openvic-simulation/map/Crime.hpp"
+#include "openvic-simulation/military/Unit.hpp"
```

Comment:
Why?

Perhaps this is a leftover from another branch?

Reply:
No, I removed an include in another place and replaced it with a forward declaration, this file just happened to rely on that removed include.